pkg/meta/version.go

```diff
@@ -3,11 +3,11 @@ package meta
 const (
 	// CLIAPIVersion used to communicate with user e.g. longhorn-manager
 	CLIAPIVersion    = 9
-	CLIAPIMinVersion = 3
+	CLIAPIMinVersion = 8
```

Comment:
8 was introduced in v1.5, so that means we only allow at least 1.5 engine cli being used in v1.6.